rosdep/python.yaml

```diff
@@ -218,6 +218,14 @@ python-babel:
     vivid: [python-babel]
     wily: [python-babel]
     xenial: [python-babel]
+python-babeltrace:
+  debian:
+    jessie_python3: [python3-babeltrace]
```

Comment:
The `<CODENAME>_python3` targets are mostly an artifact of our testing with a customized environment where python3 is changed to the default interpreter. In this case I think that the top level rule should be `python3-babeltrace` and the CODENAMES should be just the standard targets as the libraries are installable side by side.